<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Exp</class>
 <widget class="QDialog" name="Dialog_Exp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add to Experiment</string>
  </property>
  <widget class="QComboBox" name="match1Dtest2_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="match1Dtest1_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QComboBox" name="match2Dtest2_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="match1Dtest1_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="match2Dtest3_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLabel" name="match2Dtest1_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="match1Dtest3_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QLabel" name="match2Dtest2_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="match1Dtest3_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="staircase_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Staircase Tests</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shoulder_box_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="match1D_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1-D Matching Tests</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="elbow_box_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vision_box_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>vision</string>
   </property>
  </widget>
  <widget class="QComboBox" name="match2Dtest1_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="match2Dtest3_combo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>passive</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="elbow_box_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QLabel" name="match1Dtest2_lab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="match2D_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2-D Matching Tests</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>staircase_box</tabstop>
  <tabstop>shoulder_box_1</tabstop>
  <tabstop>elbow_box_1</tabstop>
  <tabstop>match1D_box</tabstop>
  <tabstop>match1Dtest1_combo</tabstop>
  <tabstop>vision_box_1</tabstop>
  <tabstop>shoulder_box_2</tabstop>
  <tabstop>elbow_box_2</tabstop>
  <tabstop>match1Dtest2_combo</tabstop>
  <tabstop>vision_box_2</tabstop>
  <tabstop>shoulder_box_3</tabstop>
  <tabstop>elbow_box_3</tabstop>
  <tabstop>match1Dtest3_combo</tabstop>
  <tabstop>vision_box_3</tabstop>
  <tabstop>shoulder_box_4</tabstop>
  <tabstop>elbow_box_4</tabstop>
  <tabstop>match2D_box</tabstop>
  <tabstop>match2Dtest1_combo</tabstop>
  <tabstop>vision_box_4</tabstop>
  <tabstop>match2Dtest2_combo</tabstop>
  <tabstop>vision_box_5</tabstop>
  <tabstop>match2Dtest3_combo</tabstop>
  <tabstop>vision_box_6</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
